README.md

```diff
@@ -19,7 +19,7 @@ Useful info and screenshots of the original SGI IRIX implementation are availabl
 
 1. Clone the repository
 2. Make a configure script: `./autogen.sh`
-3. Install dependencies (Ubuntu): `sudo apt-get install libgtk2.0-dev libgl1-mesa-dev gtkgl-2.0-dev libgtkgl2.0-dev libglu1-mesa-dev`
+3. Install dependencies (Ubuntu): `sudo apt-get install libgtk2.0-dev libgl1-mesa-dev libgtk2.0 libgtkgl2.0-dev libglu1-mesa-dev`
```

Comment:
Actually, as https://github.com/mcuelenaere/fsv/issues/15 mentions, we can just leave out the `gtkgl-2.0-dev` dependency (I just tried that in a Docker container).

Also, `libgtk2.0-dev` depends on `libgtk2.0`, so it's pointless to add it as a dependency. If you could remove it, I'll merge this PR.